calculator.c

```diff
@@ -0,0 +1,117 @@
+#include<stdio.h>
+#include<string.h>
+#include<ctype.h>
+#include<stdlib.h>
+//store number in stack
+int numStack[30];
+int topnum=-1;
+//store operators in stack
+char opStack[20];
+int topop=-1;
+//Perform arithmetic operation
+void calculate(){
+    int x,y,z,n;
```

Comment:
rename these variables apropriatly, variable/function name should be self expressive.